build.gradle.kts

```diff
@@ -12,6 +12,12 @@ plugins {
 group = "com.dlfsystems"
 version = "0.0.1"
 
+kotlin {
+    compilerOptions {
+        freeCompilerArgs.add("-Xmulti-dollar-interpolation")
```

Comment:
Needed this to have unescaped $ in my test Yegg code.  No idea why this isn't an annotation.